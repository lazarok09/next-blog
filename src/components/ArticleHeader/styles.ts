import styled, { css } from "styled-components";
import { Title as HeadingStyles } from "../Heading/styles";

export const Wrapper = styled.header`
  ${({ theme }) => css`
    padding-bottom: ${theme.spacings.xlarge};
    margin-bottom: ${theme.spacings.xlarge};
    border-bottom: 0.1rem solid ${theme.colors.mediumGray};

    ${HeadingStyles} {
      margin: 0;
      margin-bottom: ${theme.spacings.medium};
      @media ${theme.media.lteMedium} {
        font-size: calc(${theme.font.sizes.medium} + 1rem);
      }
    }
  `}
`;
export const Excerpt = styled.p`
  ${({ theme }) => css`
    margin: ${theme.spacings.medium} 0;
    font-size: ${theme.font.sizes.medium};
    @media ${theme.media.lteMedium} {
      font-size: calc(${theme.font.sizes.small} + 0.2rem);
    }
  `}
`;
export const Cover = styled.img`
  ${({ theme }) => css`
    max-width: 100%;
    display: block;
    margin-bottom: ${theme.spacings.medium};
  `}
`;
