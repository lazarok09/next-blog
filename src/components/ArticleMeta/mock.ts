import { ArticleMetaProps } from '.';

export default {
  createdAt: '2021-03-02T15:16:09.246+00:00',
  author: {
    id: 'aowea3i902103102',

    slug: 'lazaro-souza',
    displayName: 'LÃ¡zaro Souza',
  },
  categories: [
    {
      id: '123125129120',
      displayName: 'JS',
      slug: 'javascript',
    },
    {
      id: '123125129120125512',
      displayName: 'Tech',
      slug: 'tech',
    },
  ],
} as ArticleMetaProps;
