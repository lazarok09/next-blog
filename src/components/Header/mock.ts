import { HeaderProps } from '.';

export default {
  blogName: 'LÃ¡zaro',
  blogDescription:
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud.',
  logo: 'https://github.com/lazarok09.png',
  showText: true,
} as HeaderProps;
