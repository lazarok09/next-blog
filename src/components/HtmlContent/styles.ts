import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
    p {
      margin: ${theme.spacings.xlarge} 0;
    }
    a,
    a:visited,
    a:link {
      color: ${theme.colors.secondary};
      transition: all 300ms ease-in-out;
      text-decoration: none;
    }
    a:hover {
      filter: brightness(80%);
      text-decoration: underline;
    }
    pre, code{
      background: ${theme.colors.primary};
      padding: ${theme.spacings.medium};
      font-family: monospace;
      color: ${theme.colors.white};
      margin: ${theme.spacings.xlarge} 0;
      width: 100%;
      overflow-x: auto;
      font-size: ${theme.font.sizes.small};
    }
  }
    img {
        width: 100%;

    }
    .image {
      background: ${theme.colors.mediumGray};
      line-height: 0;
      margin: ${theme.spacings.xlarge} 0;
      max-width: 100%;

    }
    .image figcaption {
      font-size: ${theme.font.sizes.small};
      padding: ${theme.spacings.small};
      text-align: center;
      line-height: 1.3;
    }
    .image-style-side {
      float: right;
      max-width: 50%;
      margin: 0 ${theme.spacings.medium};
    }
    hr {
      border: none;
      border-bottom: 0.1rem solid ${theme.colors.mediumGray};
    }
    ul,
    ol {
      margin: ${theme.spacings.xlarge};
    }
    .table {
      max-width: 100%;
      overflow: hidden;
      overflow-x: auto;
    }
    table {
      width: 100%;
      border-collapse: collapse;
    }
    table th,
    table td{
      padding: ${theme.spacings.small};
      border: 0.1rem solid ${theme.colors.mediumGray};
    }
    }
    iframe {
    max-width: 100%;
    height: 50vh;
    }
    @media ${theme.media.lteMedium} {
      font-size: 2rem;

      .image-style-side {
        float: none;
        max-width: 100%;
        margin: 0;
      }
    }
  `}
`;
