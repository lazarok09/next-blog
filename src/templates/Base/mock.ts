import { BaseTemplateProps } from '.';
import { data } from '../../api/data.json';
export default {
  settings: data.setting,
  children: 'Hello World',
} as BaseTemplateProps;
