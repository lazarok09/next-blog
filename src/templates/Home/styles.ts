import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    & > img {
      height: 40px;
    }
    background: ${theme.colors.white};
  `}
`;
