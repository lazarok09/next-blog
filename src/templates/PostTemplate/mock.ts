import { PostTemplateProps } from '.';
import { data } from '../../api/data.json';
export default {
  settings: data.setting,
  post: data.posts[0],
} as PostTemplateProps;
