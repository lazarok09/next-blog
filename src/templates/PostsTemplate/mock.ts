import { data } from '../../api/data.json';
import { PostsTemplateProps } from '.';

export default {
  settings: data.setting,
  posts: data.posts,
} as PostsTemplateProps;
