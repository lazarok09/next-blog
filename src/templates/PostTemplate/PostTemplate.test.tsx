import { PostTemplate, PostTemplateProps } from '.';
import { renderTheme } from '../../styles/render-theme';
import mock from './mock';

const props: PostTemplateProps = mock;
describe('<PostTemplate />', () => {
  it('should match snapshot', () => {
    const { container } = renderTheme(<PostTemplate {...props} />);
    expect(container).toMatchSnapshot();
  });
});
