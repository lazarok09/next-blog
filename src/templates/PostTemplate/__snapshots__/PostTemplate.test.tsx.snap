// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostTemplate /> should match snapshot 1`] = `
.c19 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c19 .categories span::after {
  content: ', ';
}

.c19 .categories span:last-child::after {
  content: '';
}

.c19 a {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c19 a:hover {
  -webkit-filter: brightness(200%);
  filter: brightness(200%);
}

.c6 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c12 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: none;
}

.c16 {
  color: #000000;
  line-height: 1.2;
  font-size: 6.4rem;
  text-transform: none;
}

.c15 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c15 .c5 {
  margin: 0;
  margin-bottom: 2.4rem;
}

.c17 {
  margin: 2.4rem 0;
  font-size: 2.4rem;
}

.c18 {
  max-width: 100%;
  display: block;
  margin-bottom: 2.4rem;
}

.c21 {
  font-size: calc(1.6rem + 0.2rem);
  line-height: 1.5;
}

.c21 p {
  margin: 2.4rem 0;
}

.c21 a,
.c21 a:visited,
.c21 a:link {
  color: #dc143c;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c21 a:hover {
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c21 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c21 pre {
  background: #000000;
  padding: 2.4rem;
  font-family: monospace;
  color: #FFFFFF;
  margin: 4.0rem 0;
  margin: 2.4rem 0;
  width: 100%;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c21 pre code {
  color: inherit;
  background: inherit;
}

.c21 img {
  width: 100%;
}

.c21 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
  max-width: 100%;
}

.c21 .image figcaption {
  font-size: 1.6rem;
  padding: 1.6rem;
  text-align: center;
  line-height: 1.3;
}

.c21 .image-style-side {
  float: right;
  max-width: 50%;
  margin: 0 2.4rem;
}

.c21 hr {
  border: none;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c21 ul,
.c21 ol {
  margin: 2.4rem 4.0rem;
}

.c21 .table {
  max-width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c21 table {
  width: 100%;
  border-collapse: collapse;
  margin: 2.4rem 0;
}

.c21 table th,
.c21 table td {
  padding: 1.6rem;
  border: 0.1rem solid #DDDDDD;
}

.c21 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c21 iframe {
  max-width: 100%;
  height: 50vh;
}

.c14 {
  width: 100%;
  margin: 0 auto;
  padding: 3.2rem;
  max-width: 96rem;
}

.c20 {
  width: 100%;
  margin: 0 auto;
  padding: 3.2rem;
  max-width: 80rem;
}

.c23 {
  margin: 2.4rem 0;
}

.c23 span {
  margin: 0 0 0 0.5rem;
}

.c23 span::before {
  content: ' ';
}

.c23 span::after {
  content: ', ';
}

.c23 span:last-child::after {
  content: '';
}

.c23 a {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c23 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c25 {
  text-align: center;
  font-size: 1.6rem;
  border-top: 0.1rem solid #DDDDDD;
}

.c25 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c26 {
  position: fixed;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  z-index: 6;
  bottom: 2rem;
  right: 2rem;
  width: 4rem;
  height: 4rem;
  opacity: 0.8;
  color: #FFFFFF;
  background: #000000;
}

.c26:hover {
  opacity: 1;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.c7 > img {
  height: 15rem;
  width: 15rem;
  border-radius: 50%;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding-bottom: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  font-size: 1.6rem;
  margin: 0 auto;
}

.c10 .c5 {
  margin: 0 0 calc(1.6rem - 1rem);
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column wrap;
  -ms-flex-flow: column wrap;
  flex-flow: column wrap;
  margin-left: 3.2rem;
  max-width: 48rem;
}

.c8 {
  display: block;
  color: #FFFFFF;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin-bottom: 1.6rem;
  font-size: 1.8rem;
  border-right: 0.5rem solid #000000;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c8:hover {
  border-right: 0.5rem solid #dc143c;
  color: #dc143c;
}

.c2 {
  background: #000000;
  padding: 3.2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: fixed;
  z-index: 1;
  width: 100%;
  max-width: 32rem;
  height: 100vh;
  top: 0;
  left: 0;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  left: -31rem;
  overflow-y: hidden;
}

.c3 {
  margin: auto;
  width: 100%;
}

.c4 .c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  margin-bottom: 4.8rem;
}

.c4 .c5 img {
  border: 0.5rem solid #dc143c;
}

.c0 {
  position: fixed;
  top: 2.4rem;
  color: #FFFFFF;
  background: #000000;
  z-index: 2;
  width: 3rem;
  height: 3rem;
  left: 26rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  left: 1rem;
  color: #FFFFFF;
}

.c9 {
  margin-top: 6.4rem;
}

.c13 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  margin-bottom: 6.4rem;
}

.c24 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  padding: 3.2rem;
}

.c22 {
  padding: 0 3.2rem;
}

@media (max-width:768px) {
  .c16 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c21 {
    font-size: 2rem;
  }

  .c21 .image-style-side {
    float: none;
    max-width: 100%;
    margin: 0;
  }
}

@media (max-width:768px) {
  .c2 {
    left: -32rem;
  }
}

@media (max-width:768px) {
  .c0 {
    left: -0.5rem;
  }
}

<div>
  <div
    class=""
  >
    <a
      aria-label="Open or close menu"
      class="c0"
      href="#"
      title="Open or close menu"
    >
      <svg
        aria-hidden="true"
        aria-label="Open menu"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </a>
    <div
      aria-hidden="true"
      class="c2"
    >
      <nav
        class="c3"
      >
        <div
          class="c4"
        >
          <h1
            class="c5 c6"
          >
            <a
              class="c7"
              href="/"
              target="_self"
            >
              <img
                alt="challenge accepted"
                src="https://res.cloudinary.com/lazarok09/image/upload/v1633795031/icon_4bc14e0cf4.png"
              />
            </a>
          </h1>
        </div>
        <a
          class="c8"
          href="/"
          target="_self"
        >
          home
        </a>
      </nav>
    </div>
    <div
      class="c9"
    >
      <header
        class="c10"
      >
        <h1
          class="c5 c6"
        >
          <a
            class="c7"
            href="/"
            target="_self"
          >
            <img
              alt="challenge accepted - A blog to talk about programing and tecnology"
              src="https://res.cloudinary.com/lazarok09/image/upload/v1633795031/icon_4bc14e0cf4.png"
            />
          </a>
        </h1>
        <div
          class="c11"
        >
          <h2
            class="c5 c12"
          >
            challenge accepted
          </h2>
          <p>
            A blog to talk about programing and tecnology
          </p>
        </div>
      </header>
    </div>
    <div
      class="c13"
    >
      <article
        class=""
      >
        <div
          class="c14"
        >
          <header
            class="c15"
          >
            <h1
              class="c5 c16"
            >
              Basics JavaScript stuff
            </h1>
            <p
              class="c17"
            >
              JavaScript is a cross-platform, object-oriented scripting language. The v8 by Chrome runs the Node and make JS be a performatic language.
            </p>
            <img
              alt="Basics JavaScript stuff"
              class="c18"
              src="https://res.cloudinary.com/lazarok09/image/upload/v1633798940/photo_1560415755_bd80d06eda60_ixid_Mnwx_Mj_A3f_DB_8_M_Hxwa_G90by1w_Y_Wdlf_Hx8f_G_Vuf_DB_8f_Hx8_and_ixlib_rb_1_2_f32360cc03.jpg"
            />
            <div
              class="c19"
            >
              <p>
                <span>
                  Por 
                </span>
                <a
                  href="/author/lazaro"
                >
                  Lazaro
                </a>
                <span
                  class="separetor"
                >
                   | 
                </span>
                <time
                  datetime="2020-04-20"
                >
                  19 de abr. de 2020
                </time>
                <span
                  class="separetor"
                >
                   | 
                </span>
                <span
                  class="categories"
                >
                  <span>
                    <a
                      href="/category/javascript"
                    >
                      JavaScript
                    </a>
                  </span>
                </span>
              </p>
            </div>
          </header>
          <div
            class="c20"
          >
            <div
              class="c21"
            >
              <figure
                class="image"
              >
                <img
                  alt="white and black ceramic mug on green table cloth"
                  src="https://images.unsplash.com/photo-1585236616251-9fd0f67b0000?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80"
                />
              </figure>
              <p>
                 
              </p>
              <h2>
                 
              </h2>
              <h2>
                Introduction
              </h2>
              <p>
                JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them.
              </p>
              <p>
                JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:
              </p>
              <figure
                class="image image-style-side"
              >
                <img
                  alt="10 Wallpapers para Programadores - AlexRosa.Net | Programação, Papeis de  parede, Papeis de parede galaxia"
                  src="https://i.pinimg.com/originals/16/0b/7d/160b7d970276492f22d19f58dde2ddcb.jpg"
                />
              </figure>
              <h3>
                JavaScript and Java
              </h3>
              <p>
                    JavaScript and Java are similar in some ways but
                <strong>
                   fundamentally            different i
                </strong>
                n some others. The JavaScript language resembles Java but  does not have Java's static typing and strong type checking.            JavaScript follows most Java expression syntax, naming conventions    and basic control-flow constructs which was the reason why it was renamed from LiveScript to JavaScript.  In contrast to Java's compile-time system of classes built by  declarations, JavaScript supports a runtime system based on a small   number of data types representing numeric, Boolean, and string   values. JavaScript has a prototype-based object model instead of the more common class-based object model. The prototype-based model provides dynamic inheritance; that is, what is inherited can vary for individual objects. JavaScript also supports functions without  any special declarative requirements. Functions can be properties of objects, executing as loosely typed methods.  JavaScript is a very free-form language compared to Java. You do not    have to declare all variables, classes, and methods. You do not have  to be concerned with whether methods are public, private, or            protected, and you do not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed.  
              </p>
              <h3>
                 
              </h3>
              <h3>
                Right Text
              </h3>
              <p
                style="text-align:right;"
              >
                Functions can be properties of objects, executing as loosely typed methods.  JavaScript is a very free-form language compared to Java. You do not    have to declare all variables, classes, and methods. You do not have  to be concerned with whether methods are public, private, or            protected, and you do not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed. 
              </p>
              <h3>
                Left Text
              </h3>
              <p>
                Functions can be properties of objects, executing as loosely typed methods.  JavaScript is a very free-form language compared to Java. You do not    have to declare all variables, classes, and methods. You do not have  to be concerned with whether methods are public, private, or            protected, and you do not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed. 
              </p>
              <p>
                 
              </p>
              <h2>
                A bulleted list
              </h2>
              <ul>
                <li>
                  stuff
                </li>
                <li>
                  stuff
                </li>
                <li>
                  stuff
                </li>
                <li>
                  stuff
                </li>
              </ul>
              <blockquote>
                <h3>
                  Lázaro está citando isso pra editar um bloco de citação depois rsrs - Lázaro.
                </h3>
              </blockquote>
              <h3>
                 
              </h3>
              <h3>
                Hello world
              </h3>
              <p>
                    To get started with writing JavaScript, open the Scratchpad and            write your first "Hello world" JavaScript code:   
              </p>
              <h3>
                 
              </h3>
              <pre>
                <code
                  class="language-plaintext"
                >
                  console.log("Hello World") 
                </code>
              </pre>
              <h3>
                With Functions
              </h3>
              <pre>
                <code
                  class="language-plaintext"
                >
                    function sayHelloWorld()&lt;br /&gt;
            { return "Hello World" }
                </code>
              </pre>
              <h3>
                 
              </h3>
              <h3>
                You can build a collection of items
              </h3>
              <h4>
                Array of fruits (collection)
              </h4>
              <h4>
                <code>
                        let fruits = ["Apple", "Orange","Pinapple"]    
                </code>
                   
              </h4>
              <ul>
                <li>
                  use objects inside array itens
                </li>
              </ul>
              <p>
                  
                <code>
                  let ArrayOfObjects = [{name: "Sr.Nothing", age: 14}, {name:"Sr.OtherGuy", age: 25}]
                </code>
                 
              </p>
              <ul>
                <li>
                  Numbers are valid too  
                </li>
              </ul>
              <p>
                <code>
                   let array = [1,2,3,4,5,6,7,8,9,10] 
                </code>
              </p>
              <h2>
                Bibliotecas e Frameworks
              </h2>
              <figure
                class="table"
              >
                <table>
                  <tbody>
                    <tr>
                      <td>
                        Front-end
                      </td>
                      <td>
                        Back-end
                      </td>
                      <td>
                        Mobile
                      </td>
                      <td>
                        titutlo
                      </td>
                      <td>
                        titutlo
                      </td>
                    </tr>
                    <tr>
                      <td>
                        React
                      </td>
                      <td>
                        Node.js
                      </td>
                      <td>
                        React Native
                      </td>
                      <td>
                        info
                      </td>
                      <td>
                        info
                      </td>
                    </tr>
                    <tr>
                      <td>
                        Vue
                      </td>
                      <td>
                        Express
                      </td>
                      <td>
                        React Native
                      </td>
                      <td>
                        info
                      </td>
                      <td>
                        info
                      </td>
                    </tr>
                    <tr>
                      <td>
                        Angular
                      </td>
                      <td>
                        Nest.js
                      </td>
                      <td>
                        React Native
                      </td>
                      <td>
                        info
                      </td>
                      <td>
                        info
                      </td>
                    </tr>
                  </tbody>
                </table>
              </figure>
              <h2>
                Another Example
              </h2>
              <pre>
                <code
                  class="language-plaintext"
                >
                  var fruit = ['cherries', 'apples', 'bananas'];
fruit.sort(); // ['apples', 'bananas', 'cherries']

var scores = [1, 10, 2, 21];
scores.sort(); // [1, 10, 2, 21]
// Observe que 10 vem antes do 2,
// porque '10' vem antes de '2' em ponto de código Unicode.

var things = ['word', 'Word', '1 Word', '2 Words'];
things.sort(); // ['1 Word', '2 Words', 'Word', 'word']
// Em Unicode, números vêem antes de letras maiúsculas,
// as quais vêem antes das minúsculas.
                </code>
              </pre>
              <figure
                class="media"
              >
                <oembed
                  url="https://www.youtube.com/watch?v=PkZNo7MFNFg"
                />
              </figure>
            </div>
          </div>
        </div>
      </article>
      <div
        class="c22"
      >
        <p
          class="c23"
        >
          tags: 
           
          <span>
            <a
              href="/tag/algoritmo"
            >
              algoritmo
            </a>
          </span>
          <span>
            <a
              href="/tag/useeffect"
            >
              useEffect
            </a>
          </span>
          <span>
            <a
              href="/tag/js"
            >
              js
            </a>
          </span>
        </p>
      </div>
    </div>
    <div
      class="c24"
    >
      <footer
        class="c25"
      >
        <div
          class="c21"
        >
          <p>
            texto
          </p>
        </div>
      </footer>
    </div>
    <a
      aria-label="Go to Top"
      class="c26"
      href="#"
      title="Go to Top"
    >
      <svg
        aria-hidden="true"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
        />
      </svg>
    </a>
  </div>
</div>
`;
